### ISSUE TITLE
Ascx compilation

### ISSUE THREAD

**Issue by dazinator** [dotnet/roslyn-project-system#205]
Hi Guys, not sure if this is the correct place to create this issue.

I have created a new CPS project system, but was sad to find that .ASCX, ASPX (WebForms) files are not supported. 

This meant I had to write some significant code within my CPS based project system to handle those file types. Basically, parsing webforms markup, and generating appropriate `.designer.cs` (code behind) files.

I suspect I know the answer to this, but would you not consider also updating the old ASP.NET WebForms project system to be CPS / Roslyn based, as I am sure you are aware WebForms projects are likely to remain in the wild for sometime (legacy applications) and the tooling / project system could do with some love.

**Comment by davkean** [comment]
You are correct, there's zero tooling right now to handle these files in CPS. When/if we bring the "Web Application Project" to CPS, then that tooling will come a long with it and light up, but until then unfortunately, you'll have to write it yourself. We don't have a timeline or plan for that yet.
